edx-platform/bp-theme/lms/templates/course.html

```diff
@@ -0,0 +1,49 @@
+<%def name="online_help_token()"><% return "course" %></%def>
+<%namespace name='static' file='static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.urls import reverse
+from six import text_type
```

Comment:
Please, @amalbas Sort the imports according to [PEP8](https://www.datacamp.com/community/tutorials/pep8-tutorial-python-code#import).